### ISSUE TITLE
Assemble Release Problem

### ISSUE THREAD

**Issue by eskiesirius** [lvlrSajjad/react-native-launch-application#3]
Hello i am having a problem when releasing..

"react": "16.11.0"
"react-native": "0.62.2"

i got this error...
```
Execution failed for task ':react-native-launch-application:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource linking failed
     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\ab2c0d53782effd02ac7f499ae154019\appcompat-1.0.2\res\values-v26\values-v26.xml:5:5-8:13: AAPT: error: resource android:attr/colorError not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\ab2c0d53782effd02ac7f499ae154019\appcompat-1.0.2\res\values-v26\values-v26.xml:9:5-12:13: AAPT: error: resource android:attr/colorError not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\ab2c0d53782effd02ac7f499ae154019\appcompat-1.0.2\res\values-v26\values-v26.xml:13:5-16:13: AAPT: error: style attribute 'android:attr/keyboardNavigationCluster' not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\ab2c0d53782effd02ac7f499ae154019\appcompat-1.0.2\res\values-v28\values-v28.xml:5:5-8:13: AAPT: error: resource android:attr/dialogCornerRadius not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\ab2c0d53782effd02ac7f499ae154019\appcompat-1.0.2\res\values-v28\values-v28.xml:9:5-12:13: AAPT: error: resource android:attr/dialogCornerRadius not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:57:5-88:25: AAPT: error: resource android:attr/fontStyle not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:57:5-88:25: AAPT: error: resource android:attr/font not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:57:5-88:25: AAPT: error: resource android:attr/fontWeight not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:57:5-88:25: AAPT: error: resource android:attr/fontVariationSettings not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:57:5-88:25: AAPT: error: resource android:attr/ttcIndex not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:89:5-125:25: AAPT: error: resource android:attr/startX not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:89:5-125:25: AAPT: error: resource android:attr/startY not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:89:5-125:25: AAPT: error: resource android:attr/endX not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:89:5-125:25: AAPT: error: resource android:attr/endY not found.

     C:\xxxxxx\.gradle\caches\transforms-2\files-2.1\6e813fa6b8fc8b59c3519258210e2a27\core-1.0.1\res\values\values.xml:126:5-132:25: AAPT: error: resource android:attr/offset not found.
```

**Comment by Yatanvesh** [comment]
try modifying the package's build.gradle values to match your own project's build.gradle file. you can change compileSDkversion, target and minSDKVersion to match your project and it'll compile. If it works, you can fork the package and open a pr or point npm to your own github repo

**Comment by eskiesirius** [comment]
I just created my own launch application by copying the native code and now it is ok

**Comment by fachrezamuslim** [comment]
i got same issue, any update for this?

**Comment by eskiesirius** [comment]
> i got same issue, any update for this?

Just create your own.. just copy the codes..

**Comment by fachrezamuslim** [comment]
> > i got same issue, any update for this?
> 
> Just create your own.. just copy the codes..

i'm not familiar with java, which codes should i copy? these codes?

SajjadLaunchApplicationPackage.java
SajjadLaunchApplicationModule.java

**Comment by eskiesirius** [comment]
> > > i got same issue, any update for this?
> > 
> > 
> > Just create your own.. just copy the codes..
> 
> i'm not familiar with java, which codes should i copy? these codes?
> 
> SajjadLaunchApplicationPackage.java
> SajjadLaunchApplicationModule.java

yes.. then add the package in your MainApplication.. and dont forget to add the ApplicationModule in your manifest.. 
actually this is also my first time doing native..

**Comment by fachrezamuslim** [comment]
thanks @eskiesirius . 

btw, why we need to add ApplicationModule in manifest ? i follow https://reactnative.dev/docs/native-modules-android instructions. it didn't say to put ApplicationModule in manifest

**Comment by eskiesirius** [comment]
> thanks @eskiesirius .
> 
> btw, why we need to add ApplicationModule in manifest ? i follow https://reactnative.dev/docs/native-modules-android instructions. it didn't say to put ApplicationModule in manifest

oh.. as i said.. it was my first time using native code.. haha you can try, im not sure about it though... so if it didn't work.. you can add it in your manifest..

**Comment by fachrezamuslim** [comment]
@eskiesirius  have you test the module with your devices? 

the module working good in my emulator (can open from background or from inactive state), but in some devices like Xiaomi (can't open from background or inactive state), and Oppo (can't open from inactive state)

**Comment by eskiesirius** [comment]
> @eskiesirius have you test the module with your devices?
> 
> the module working good in my emulator (can open from background or from inactive state), but in some devices like Xiaomi (can't open from background or inactive state), and Oppo (can't open from inactive state)

I am using xiaomi.. i only use this in my incoming call function.. the only problem is that when I close my app.. incoming call function is not working but launching the app works fine..

**Comment by fachrezamuslim** [comment]
> > @eskiesirius have you test the module with your devices?
> > the module working good in my emulator (can open from background or from inactive state), but in some devices like Xiaomi (can't open from background or inactive state), and Oppo (can't open from inactive state)
> 
> I am using xiaomi.. i only use this in my incoming call function.. the only problem is that when I close my app.. incoming call function is not working but launching the app works fine..

what is your android version?

my Xiaomi using Android 10 and my Oppo using Android 8

**Comment by eskiesirius** [comment]
> > > @eskiesirius have you test the module with your devices?
> > > the module working good in my emulator (can open from background or from inactive state), but in some devices like Xiaomi (can't open from background or inactive state), and Oppo (can't open from inactive state)
> > 
> > 
> > I am using xiaomi.. i only use this in my incoming call function.. the only problem is that when I close my app.. incoming call function is not working but launching the app works fine..
> 
> what is your android version?
> 
> my Xiaomi using Android 10 and my Oppo using Android 8

I am using Android 9

**Comment by fachrezamuslim** [comment]
> > > > @eskiesirius have you test the module with your devices?
> > > > the module working good in my emulator (can open from background or from inactive state), but in some devices like Xiaomi (can't open from background or inactive state), and Oppo (can't open from inactive state)
> > > 
> > > 
> > > I am using xiaomi.. i only use this in my incoming call function.. the only problem is that when I close my app.. incoming call function is not working but launching the app works fine..
> > 
> > 
> > what is your android version?
> > my Xiaomi using Android 10 and my Oppo using Android 8
> 
> I am using Android 9

hmm, i test the module with Samsung (Android 9) and its working. i can open my app from background and inactive state. this is weird

**Comment by Yatanvesh** [comment]
Ill pick up the issue and open a PR with updated config for this package, @fachreza73  @eskiesirius will that be helpful?

**Comment by eskiesirius** [comment]
sure..

**Comment by amashabe** [comment]
> I just created my own launch application by copying the native code and now it is ok

can you share that code with us?
